pandas/core/generic.py

```diff
@@ -10464,7 +10458,7 @@ def _doc_parms(cls):
 %(name2)s.cumsum : Return cumulative sum over %(name2)s axis.
 %(name2)s.cumprod : Return cumulative product over %(name2)s axis.
 
-%(examples)s
+%(examples)s\
```

Comment:
What's this needed for?

Reply:
Yeah, I'm not entirely sure re: this change, but it resolves the following `GL03` errors:

```
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Series.cummax: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Series.cummin: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Series.cumprod: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Series.cumsum: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Panel.cummax: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Panel.cummin: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Panel.cumprod: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.Panel.cumsum: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.DataFrame.cummax: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.DataFrame.cummin: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.DataFrame.cumprod: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
##vso[task.logissue type=error;sourcepath=pandas/core/generic.py;linenumber=11015;code=GL03;]pandas.DataFrame.cumsum: Double line break found; please use only one blank line to separate sections or paragraphs, and do not leave blank lines at the end of docstrings
```

The only other approaches I can see are to do the same thing on the individual `_cummax/min/prod/sum_examples` strings or to `rstrip()` the doc in the `Appender` decorator. Any ideas?

Reply:
What if you just moved the triple quotes up a line instead of using the forward slash? I guess this occurs because all of the Examples already have a newline at the end so this doubles up as is

Reply:
Ah, thanks — I thought that would have introduced a lint error.